apps/admin/preview/base/kustomization.yaml

```diff
@@ -2,11 +2,21 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
   - ../../base
-  - kube-slack-values.yaml
-  - external-dns.yaml
-  - manage-configmaps-clusterrole.yaml
+  - ../../keda
+  - ../../traefik2
+  - ../../traefik2/preview/secret-provider-internal.yaml
+  - ../../external-dns-2/
+  - ../../aad-pod-id/mi
+  - ../../secrets-csi-driver-crds
+  - ../../secrets-csi-driver
   - ../../../rbac/edit-clusterrole.yaml
   - ../../../rbac/reader-clusterrole.yaml
-  - ../../keda
+  - kube-slack-values.yaml
+  - manage-configmaps-clusterrole.yaml
+
 patchesStrategicMerge:
-  - ../../aat/base/keda-identity.yaml
\ No newline at end of file
+  - ../../aat/base/keda-identity.yaml
+  - ../../traefik2/preview/secret-provider.yaml
+  - ../../traefik2/preview/tlsstore-patch.yaml
```

Comment:
Is this missing 
`../../aad-pod-id/mi/preview/azure-identity-patch.yaml`

Reply:
Does it need more adding than `- ../../aad-pod-id/mi` in `/base` @thomast1906 ?

Reply:
https://github.com/hmcts/cnp-flux-config/tree/master/apps/admin/aad-pod-id/mi/preview ?

This was here previous https://github.com/hmcts/cnp-flux-config/pull/16873/files#diff-75725a845e4c07616bdfbbd8093fb421b149d92aa662a0f14dc677ef9caab870L16 

Can we doublecheck all patch removals etc are included in the base

Reply:
Added now and base here looks the same as it was in previous patch minus a cluster specific file